TS01-Time-Series_Metadata.Rmd

```diff
@@ -65,37 +60,20 @@ refresher on setting the working directory, please view the <a href="XXX" target
 
 </div>
 
-#Understand Your Data 
```

Comment:
Yes, because it was in Lesson 00. I see how it is tricky to balance for online vs workshop. I was thinking as if someone was working through all lessons online and finding it repetitive. If we expect folks to skip around, I would also expect them to realize they are skipping around and might need to look at prior lessons to get the full scope. I defer to you and NEON on that one!